applications/ShapeOptimizationApplication/tests/.gitignore

```diff
@@ -0,0 +1,7 @@
+**/Optimization_Results/
+
+**/response_combination.csv
+
+**/Optimization_Results*/
+**/algorithm_gradient_projection_test
```

Comment:
There is no such test case. The same holds for the following linen.

Reply:
accidentally commited